no-computer.md

```diff
@@ -2,54 +2,122 @@
+_Why was this DSL appropriate for its users? How does it address the need?_
+
+The abacus is extremely appropriate for its users. Users in the times before did
+not need to learn how to read to use the abacus, they just needed to understand
+how to do arithmetic. The abacus helped them do the calculations and keep track
+of the results.
 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
+_Why do you think this is a DSL?  How did you come up with it?_
 
+I was thinking about what I liked to do and how things were taught and how
+people looked at it back home in China. I really like doing math and remembered
+that abacus was a popular extra-curricular that kids did that I did not do! I
+did not do worse than the kids that learned the abacus in arithmetic. However,
+after thinking about it thoroughly, the abacus definitely reinforces your brain
+in calculating numbers and so forth. Its uses in the before-calculator times are
+relatively obvious.
 
 ### Domain
-_Describe the project's domain in five words._
+_Describe the DSL's domain in five words._
 
+Speedy calculation of multiple numbers.
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
-
+_How might the user interact with the language? How does using it look
+like? Why is this the right way to interact with the problem domain?_
+
+The user interacts it by moving the different beads on the abacus. Depending on
+the abacus, different beads refer to different things. Usually, each bead refers
+to one of some unit. So you could have the right-most rod have beads the refer
+to the 1s, where one bead on the bottom of the rod means 1, and the two beds on
+the top row of that rod mean 5 each. Thus it covers the whole spectrum of 10.
+The rod to its left would be for the 10s place, each lower row bead being 10,
+with a top row bead for that rod being 50. When adding, you move the correct
+beads that represent the numbers being added, etc.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The abacus doesn't do anything to/for the user except keep numbers as is so that
+the user can think about other things in the mean time. The only kind of errors
+that can occur are human calculation errors. For example, if someone puts in the
+wrong number at the beginning, no amount of calculating will fix the siatuoin
+until the user figures out they put in the wrong number.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Calculating numbers and not changing the number on the abacus until someone
+touches it shoudl be easy. More complex arithmetic/mathematical operations like
```

Comment:
When you say "not changing the number on the abacus until someone touches it", do you mean using the abacus not only as a calculator, but also as a kind of memory device? Isn't that a little outside of the abacus's domain? For example, there are other "DSLs" for writing down numbers on paper or some other medium, so that they can be stored.

I agree with your other points here though, they seem to be very much in line with what the abacus is/was used for.